sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/Internals/AzureMonitorTransmitter.cs

```diff
@@ -108,15 +109,9 @@ public async ValueTask TransmitFromStorage(long maxFilesToTransmit, bool async,
                 try
                 {
                     // TODO: Do we need more lease time?
-                    var blob = _storage.GetBlob()?.Lease(10000);
-                    if (blob == null)
+                    if (_storage.TryGetBlob(out var blob) && blob.TryLease(1000))
```

Comment:
What if `blob` value is null?

Reply:
Then tryget will return false - https://github.com/open-telemetry/opentelemetry-dotnet-contrib/blob/00ac3601a6f85ed44840641a0d40c8f122732e79/src/OpenTelemetry.Extensions.PersistentStorage/FileBlobProvider.cs#L137